python/lsst/ts/observatory/control/base_calsys.py

```diff
@@ -26,6 +26,7 @@
 import logging
 import typing
 
+import jsonschema
```

Comment:
Instead of using `jsonschema` directly, I would recommend you use `salobj.DefaultingValidator` as it allows you to add default values for the parameters, reducing the amount of duplicated needed in the configuration.

This is what we use for [SAL Scripts](https://github.com/lsst-ts/ts_salobj/blob/4841de22e72e6e8bc95442d289e9afafa7bb248e/python/lsst/ts/salobj/base_script.py#L116).

You can check the link above to see how we use it.